src/server/routes/monthData.js

```diff
@@ -86,24 +87,23 @@ function selectMonthData(req, res, next) {
   // Checks if the URL input fits season's convention
   const seasonMatches = matchListDate(dateRequest);
   // Default audience list is the adult list
-  let audience = 'Adult';
+  let audience = '';
   let isValidAudience = true;
   let requestedSeason = '';
 
   // Checks if req.query.audience exists
   if (req.query.audience) {
+    audience = req.query.audience;
     // If so, checks if it equals to one of the three values
-    if (['Adult', 'YA', 'Children'].includes(req.query.audience)) {
-      // If so, updates the selected audience list value
-      audience = req.query.audience;
-    } else {
-      // Or set it is an invalid audience query
-      isValidAudience = false;
-    }
+    isValidAudience = ['Adult', 'YA', 'Children'].includes(audience);
   }
 
   if (!seasonMatches || !isValidAudience) {
-    logger.error(`Status Code: 400, Error Message: Invalid season ${seasonMatches[0]} or audience ${audience}, Redirecting to: ${config.baseUrl}/404`);
+    let matches = 'undefined';
```

Comment:
Why do you have to make this a string?

Reply:
Was trying to assign as undefined, but I can probably just assign no value instead which will result in undefined.